provenance/_config.py

```diff
@@ -5,6 +5,11 @@
 import copy
 import toolz as t
 
+try:
+    import provenance.sftp as sftp
+except ImportError:
```

Comment:
I would move this down to when you try to add it to the dict... so:

```python
try:
    import provenance.sftp as sftp
    BLOBSTORE_TYPES['sftp'] = sftp.SFTPStore
except ImportError:
    pass
```

I wouldn't worry about printing a message for now. Ideally, an exception would be thrown when they tried to use the SFTP store.. which I guess you could do something like:

```python
try:
    import provenance.sftp as sftp
    BLOBSTORE_TYPES['sftp'] = sftp.SFTPStore
except ImportError as e:

    class SFTPStore(object):
        def __init__(*args, **kargs):
            raise(e)

    BLOBSTORE_TYPES['sftp'] = SFTPStore
```